Windows/DifficultyPredictorWindow.xaml.cs

```diff
@@ -136,10 +138,10 @@ public partial class DifficultyPredictorWindow : Window {
         }
 
         private bool CheckParameterRanges(double noteDensity, double averageTimeDifference, int longestHighDensitySequence, double peakNoteDensity) {
-            return Helper.DoubleRangeCheck(noteDensity, 1.627074, 7.615101) &&
```

Comment:
should probably put the magic numbers in the constants file

Reply:
Done